snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
@kyrofa if this gets annoying btw it can be done in a follow by you or somebody else, don't think it should block this